include/event_sources/timer.h

```diff
@@ -27,23 +43,31 @@ class Timeout : public microloop::EventSource
 
     set_fd(fd);
 
-    itimerspec timer_value{{0, 0}, {timeout / 1000, timeout % 1000 * 1000000}};
+    timespec t{value / 1000, value % 1000 * 1000000};
```

Comment:
For high accuracy, high resolution timer is better.

Reply:
I am not sure such precision is really needed when setting a timeout or an interval timer. The `set_timeout` function guarantees that for a Timer with value X, the callback will be triggered after **approximately** X milliseconds + any delay `epoll` might create before it can process the event coming from `timerfd`. The same applies for interval timers.